specification/deviceregistry/resource-manager/readme.md

```diff
@@ -71,7 +71,6 @@ This is not used by Autorest itself.
 swagger-to-sdk:
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-java
-  - repo: azure-sdk-for-python
```

Comment:
Please help remove `- repo: azure-sdk-for-python` of readme.md to disable generation from swagger if we already decide to generate from Typepsec.

Reply:
got it